sourcing/module.go

```diff
@@ -0,0 +1,28 @@
+package sourcing
+
+import (
+	"flamingo.me/dingo"
+
+	"flamingo.me/flamingo-commerce/v3/cart"
+	"flamingo.me/flamingo-commerce/v3/sourcing/application"
+	"flamingo.me/flamingo-commerce/v3/sourcing/domain"
+)
+
+type (
+	// Module registers sourcing module
+	Module struct {
+	}
+)
+
+// Configure module
+func (m *Module) Configure(injector *dingo.Injector) {
+	injector.Bind(new(domain.SourcingService)).To(domain.DefaultSourcingService{})
```

Comment:
Do we want to Bind the `DefaultSourcingService` always? This means a special implementation has to overwrite the binding.

Reply:
Yes, I think we really wanted to have this behaviour to have the module running almost flawless.

Reply:
Made it configurable